### PR TITLE
[QRF-113] Fix - Iframely Embeds not visible on first page load

### PR DESCRIPTION
**Pull request by e1himself**

### CODE REVIEW

File: src/components/HtmlInjection.tsx [hunk 1]
```diff
@@ -1,28 +1,49 @@
...
+        ).map((script) => {
+            return Array.from(script.attributes).reduce((agg, { name, value }) => ({ ...agg, [name]: value }), {});
+        });
+
+        container.remove();
+
+        scripts.forEach((attributes) => {
+            if (attributes.src && document.querySelector(`script[src="${attributes.src}"]`)) {
+                return;
+            }
```
Comment (e1himself) [inline src/components/HtmlInjection.tsx:32]: This prevents including the same scripts again and again, which causes problems with content initialization.

File: src/components/HtmlInjection.tsx [hunk 2]
```diff
@@ -1,28 +1,49 @@
...
+
+            const script = document.createElement('script');
+            setScriptAttributes(script, attributes);
+
+            script.addEventListener('error', onError);
+            document.body.appendChild(script);
+        });
+
+        iframely?.load();
```
Comment (e1himself) [inline src/components/HtmlInjection.tsx:41]: This additional call will make sure iframely catches up new dynamically loaded embeds when they appear.

File: src/elements/Embed/IframelyEmbed.scss [hunk 3]
```diff
@@ -9,4 +9,8 @@
         border: 1px solid $color-error;
         border-radius: $border-radius-base;
     }
+
+    blockquote {
+        border-left: none;
+    }
```
Comment (e1himself) [inline src/elements/Embed/IframelyEmbed.scss:15]: There was an issue with displaying TikTok embeds, which had a blockquote wrapper. The block had unwanted left border inherited from global styles.

File: src/components/Media/Media.tsx [hunk 4]
```diff
@@ -34,7 +34,7 @@ export function Media({ className, image, style, title }: Props) {
                 playsInline
                 style={style}
                 title={title}
-                webkit-playsinline
+                webkit-playsinline="true"
```
Comment (e1himself) [inline src/components/Media/Media.tsx:37]: React was complaining about boolean value being used for a non-boolean attribute.

**Review by aspirisen — APPROVED** [review]
👍

**Pull request merged**